modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/EntryStatusDataRenderer.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React, {useEffect, useState} from 'react';
+
+import '../../../../object_entries/ViewObjectEntries.scss';
```

Comment:
Why not create the `ObjectEntryStatusDataRenderer.scss`?

Reply:
we can do that, I thought this file could be used in the future in case we needed to add other styles to view object entries, but I will do that!